styles/globals.css

```diff
@@ -26,6 +26,11 @@
   border: none;
 }
 
+.modal-header {
+  background-color: var(--bs-blue);
+  color: white;
+}
+
```

Comment:
Adding this removes the 1px white border that sort of shows up on chrome in windows:

```
.modal-content {
  border: none;
}
```